spec/app/models/metasploit_data_models/module_run_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'spec_helper'
+
+describe MetasploitDataModels::ModuleRun do
+
+  subject(:module_run){FactoryGirl.build(:metasploit_data_models_module_run)}
+
+  context "database columns" do
+    it { is_expected.to have_db_column(:attempted_at).of_type(:datetime) }
+    it { is_expected.to have_db_column(:fail_detail).of_type(:text) }
+    it { is_expected.to have_db_column(:fail_reason).of_type(:string) }
+    it { is_expected.to have_db_column(:module_detail_id).of_type(:integer) }
+    it { is_expected.to have_db_column(:module_full_name).of_type(:text) }
+    it { is_expected.to have_db_column(:port).of_type(:integer) }
+    it { is_expected.to have_db_column(:proto).of_type(:string) }
+    it { is_expected.to have_db_column(:session_id).of_type(:integer) }
+    it { is_expected.to have_db_column(:status).of_type(:string) }
+    it { is_expected.to have_db_column(:trackable_id).of_type(:integer) }
+    it { is_expected.to have_db_column(:trackable_type).of_type(:string) }
+    it { is_expected.to have_db_column(:user_id).of_type(:integer) }
+    it { is_expected.to have_db_column(:username).of_type(:string) }
+  end
+
+  context "associations" do
```

Comment:
Non-polymorphic associations should be tested that they declare `inverse_of` as it ensures that both sides of the association remain consistent in memory.